spark8t/cli/service_account_registry.py

```diff
@@ -42,6 +47,18 @@ class Actions(str, Enum):
     LIST = "list"
 
 
+def create_namespace_if_missing(kube_interface: AbstractKubeInterface, namespace: str):
+    """Create namespace if does not exist."""
+    if not kube_interface.exists(KubernetesResourceType.NAMESPACE, namespace):
+        try:
+            kube_interface.create(KubernetesResourceType.NAMESPACE, namespace)
+        except Exception:
+            logger.warn(
+                f"Namespace {namespace} does not exist, and it could not be created."
+            )
+            raise NamespaceNotFound(namespace)
```

Comment:
nit: In my opinion this class name doesn't truly describe that the namespace could not be created (which is the case because `Exception` catches everything that is thrown during the creation of namespace).